src/WebJobs.Script.WebHost/Filters/AuthorizationLevelAttribute.cs

```diff
@@ -100,6 +101,26 @@ internal static AuthorizationLevel GetAuthorizationLevel(HttpRequestMessage requ
             return AuthorizationLevel.Anonymous;
         }
 
+        internal static bool IsValidKey(string keyToValidate, string functionKey)
```

Comment:
It's not know yet to be multiple keys, so I just left the name to match the model property type